src/runmodes.c

```diff
@@ -188,9 +190,9 @@ char *RunmodeGetActive(void)
  *
  * \return a string containing the current running mode
  */
-const char *RunModeGetMainMode(void)
+const char *RunModeGetMainMode(int index)
```

Comment:
I'm a bit confused by the use of 'RunModeGetMainMode' and 'RunmodeGetPrimary'. Do we need both?